Adyen.Test/mockHttpClient.cs

```diff
@@ -0,0 +1,34 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class MockHttpMessageHandler : HttpMessageHandler
+{
+    private readonly string _response;
+    private readonly HttpStatusCode _statusCode;
+
+    public string Input { get; private set; }
+    public int NumberOfCalls { get; private set; }
```

Comment:
Is the `NumberOfCalls` used ?

Reply:
No, I used this mock based on an article on internet. If we want to include more HTTP client tests in the future we would be able to use this field. Or I can just remove it for now.

Reply:
It is a mock so it is ok